docs/configuration.md

```diff
@@ -177,9 +177,20 @@ afterEach(() => {
 })
 ```
 
+### `safeDescriptors: boolean = true`
+
+MobX makes some fields **non**-`configurable` or **non**-`writable` to prevent you from doing things that are not supported or would most likely break your code. However this can also prevent spying/mocking/stubbing in your tests.
+`configure({ safeDescriptors: false })` disables this safety measure, making everything `configurable` and `writable`.
+Note it doesn't affect existing observables, only the ones created after it's been configured.
+<span style="color:red">Use with caution</span> and only when needed - do not turn this off globally for all tests, otherwise you risk false positives (passing tests with broken code).
```

Comment:
I think the last sentence is a little pedantic, and I'd argue it is actually quite safe to turn it off for unit tests, as in all other cases it will fail fast, which should be covered by any superficial form of integration, e2e, manual testing

Reply:
It may fail as slow as anything else (on specific IO), especially in case of non-writable actions on plain (dynamics) objects.
Redefinition in class is not the only case where it can fail, if that's what you're assuming - and you still have to create an instance, which can happen absolutely anytime.

Reply:
We can always be less pedantic. For now lets be cautious, so if there is a problem, we can always say: *"We told you so"* :)